RELEASES.md

```diff
@@ -1,3 +1,202 @@
+Version 1.26.0 (2018-05-10)
+==========================
+
+Language
+--------
+- [Closures now implement `Copy + Clone` if all captured variables do.][49299]
+- [Stabilised the `!` type.][47630] (Also known as the never type).
+- [The inclusive range syntax e.g. `for x in 0..=10` is now stable.][47813]
+- [`impl Trait` is now stable allowing you to have abstract
+  return types.][49255] e.g. `fn foo() -> impl Iterator<Item=u8>` or
+  `fn open(path: impl AsRef<Path>)`.
+- [Pattern matching on references are now automatically dereferenced.][49394]
+- [128-bit integers in the form of `u128` & `i128` are now stable.][49101]
```

Comment:
I think “`u128` and `i128`” is better than “`u128` & `i128`”, both from a grammar (English not Rust) point of view and because the ampersand can be confused with the `&` operator at first glance.